src/snowflake/snowpark/_internal/packaging_utils.py

```diff
@@ -483,14 +513,12 @@ def add_snowpark_package(
         channel.
 
     Raises:
-        pkg_resources.DistributionNotFound: If the Snowpark Python Package is not installed in the local environment.
+        importlib.metadata.PackageNotFoundError: If the Snowpark Python Package is not installed in the local environment.
```

Comment:
nit: do we need this in the change log for this error behavior change?

Reply:
This is an internal API. But we do need to check if any public APIs expose it.